lib/screens/tabs_screen.dart

```diff
@@ -40,9 +39,31 @@ class _TabsScreenState extends State<TabsScreen> {
           Builder(builder: (context) {
             return IconButton(
                 onPressed: () async {
-                  GoogleSignIn googleSignIn = GoogleSignIn();
-                  await googleSignIn.disconnect();
-                  await FirebaseAuth.instance.signOut();
+                  showDialog(
+                      context: context,
+                      builder: (_) {
+                        return AlertDialog(
+                          title: Text("Confirm Sign Out?"),
+                          actions: [
+                            ElevatedButton(
+                              onPressed: () async{
+                                GoogleSignIn googleSignIn = GoogleSignIn();
+                                await googleSignIn.disconnect();
```

Comment:
This line is returning an exception when the sign in method is not Google. Figure out if there is a way to find the sign in method, or just add this line in try catch block.

Reply:
I have corrected it now. Please review.